src/core/send_buffer.c

```diff
@@ -46,7 +46,7 @@
     streams, and its legacy APIs don't let it pass the problem further up the
     stack to the layer that really knows.
 
-    So then, we indicate min(SendBuffer.IdealBytes, Stream.SendWindow) as the
+    So then, we indicate CXPLAT_MIN(SendBuffer.IdealBytes, Stream.SendWindow) as the
```

Comment:
Can you go through and either revert the comment changes or fix up the line lengths?